pkgs/data/themes/cdetheme/default.nix

```diff
@@ -11,16 +15,26 @@ stdenv.mkDerivation rec {
     sha256 = "1v5c4db69cmzdci8xxlkx3s3cifg1h5160qq5siwfps0sj7pvggj";
   };
 
-  dontBuild = true;
+  dontInstall = true;
 
-  pythonPath = with python2Packages; [ pyqt4 pillow pyxdg pyyaml ];
-  nativeBuildInputs = with python2Packages; [ python wrapPython ];
+  buildInputs = [
+    (python2.withPackages (pythonPackages: with pythonPackages; [
+      pyqt4
+      pillow
+      pyxdg
+      pyyaml
+    ]))
+    xvfb_run
+  ];
 
-  installPhase = ''
-    mkdir -p $out/share/themes
-    cp -r cdetheme $out/share/themes
-    patchShebangs $out/share/themes/cdetheme/scripts/switchtheme
-    wrapPythonProgramsIn "$out/share/themes/cdetheme/scripts" "$out $pythonPath"
+  buildPhase = ''
+    export HOME=`pwd`
+    cd cdetheme/scripts
+    substituteInPlace switchtheme \
+      --replace "Globals.themesrcdir=Globals.themedir" "Globals.themesrcdir='$HOME/cdetheme'" \
+      --replace "Globals.themedir=os.path.join(userhome,'.themes','cdetheme')" "Globals.themedir='$out/share/themes/cdetheme'" \
```

Comment:
I am not sure here but shouldn't $out be evaluated here?

Reply:
@SuperSandro2000 Sorry, don't get. It is evaluated here like expected